code/modules/mob/living/simple_animal/abnormality/waw/despair_knight.dm

```diff
@@ -131,18 +124,56 @@
 		blessed_human = user
 		RegisterSignal(user, COMSIG_LIVING_DEATH, .proc/BlessedDeath)
 		RegisterSignal(user, COMSIG_HUMAN_INSANE, .proc/BlessedDeath)
-		to_chat(user, "<span class='nicegreen'>You feel protected.</span>")
-		user.physiology.red_mod *= 0.5
-		user.physiology.white_mod *= 0.5
-		user.physiology.black_mod *= 0.5
-		user.physiology.pale_mod *= 2
-		user.add_overlay(mutable_appearance('ModularTegustation/Teguicons/tegu_effects.dmi', "despair", -MUTATIONS_LAYER))
-		playsound(get_turf(user), 'sound/abnormalities/despairknight/gift.ogg', 50, 0, 2)
+		user.apply_status_effect(STATUS_EFFECT_DESPAIR_BLESSING)
 	return
 
 /mob/living/simple_animal/hostile/abnormality/despair_knight/breach_effect(mob/living/carbon/human/user)
 	..()
+	if(!isnull(blessed_human))
+		UnregisterSignal(blessed_human, COMSIG_LIVING_DEATH)
+		UnregisterSignal(blessed_human, COMSIG_HUMAN_INSANE)
+		blessed_human.remove_status_effect(STATUS_EFFECT_DESPAIR_BLESSING)
+		blessed_human = null
 	icon_living = "despair_breach"
 	icon_state = icon_living
 	addtimer(CALLBACK(src, .proc/TryTeleport), 5)
 	return
+
+/datum/status_effect/despair_blessed
+	id = "despair_bless"
+	status_type = STATUS_EFFECT_UNIQUE
+	duration = -1
+	alert_type = /atom/movable/screen/alert/status_effect/despair_blessed
+
+/atom/movable/screen/alert/status_effect/despair_blessed
```

Comment:
The amount of screen alerts will be ennormous by that point, might as well remove it from here since it already puts a visible icon next to your character.